doc/examples/functional_usage.rst

```diff
@@ -0,0 +1,39 @@
+Using Renate Functionally
+*************************
+
+Usually, we use Renate by writing a :code:`renate_config.py` and launching training jobs via the
+:py:func:`~renate.training.training.run_training_job` function. In this example, we demonstrate how
+to write your own training script and use renate in a functional way. This can be useful, e.g., for
+debugging new components.
+
+Here, we use Renate to fine-tune a pretrained Transformer model on a sequence classification
+dataset. First, we create the model and a loss function. Since this is a static model, we simply
+wrap it using the :py:class:`~renate.models.renate_module.RenateWrapper` class.
+
+.. literalinclude:: ../../examples/functional_usage/train.py
+    :lines: 12-16
```

Comment:
if reduction = none is required, you need to explain why

Reply:
Done.